sample-applications/fili-wikipedia-example/README.md

```diff
@@ -18,6 +18,7 @@ This example is an entirely self contained example that provides a Fili applicat
     ```bash
     cd fili
     mvn install
+    cd sample-applications
```

Comment:
without this, the terminal gives an error at `mvn -pl fili-wikipedia-example exec:java` complaining a wrong directory